pandas/tests/io/formats/style/test_style.py

```diff
@@ -1547,3 +1547,10 @@ def test_col_trimming_hide_columns():
         assert ctx["head"][0][c + 2]["is_visible"] == vals[1]
 
     assert len(ctx["body"][0]) == 6  # index + 2 hidden + 2 visible + trimming col
+
+
+@pytest.mark.parametrize("format", ["html", "latex", "string"])
```

Comment:
```suggestion
@pytest.mark.parametrize("format", ["html", "latex"])
```
`to_string` is a new method in 1.5, hence this fails